libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/NavigationInstructionPlayer.java

```diff
@@ -72,8 +79,11 @@ private void initAudioFocusRequest() {
   }
 
   private void initInstructionPlayers(Context context, String language, String accessToken) {
-    mapboxSpeechPlayer = new MapboxSpeechPlayer(context, language, accessToken);
-    mapboxSpeechPlayer.setInstructionListener(this);
+    if (languageSupportedByMapboxSpeech) {
```

Comment:
What about extracting the init of `mapboxSpeechPlayer` and `androidSpeechPlayer` into their own methods?